packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -288,6 +287,24 @@ class Checkout extends Component<
 
             window.addEventListener('beforeunload', this.handleBeforeExit);
 
+            const isSentryLoggingAll =
+                data.getConfig()?.checkoutSettings.features[
+                    'CHECKOUT-7764.Increase_sentry_logging_to_100_percent'
+                ] ?? false;
+
+            if (isSentryLoggingAll) {
+                const { sentryConfig, publicPath } = this.props;
+
+                this.errorLogger = createErrorLogger(
```

Comment:
According to the Sentry doc, changing the error sample rate requires re-deployment.
https://docs.sentry.io/platforms/php/guides/laravel/configuration/sampling/#sampling-error-events